src/errorHandler.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Alert, Clipboard } from 'react-native';
+import hathorLib from '@hathor/wallet-lib';
+import * as Sentry from '@sentry/react-native';
+import VersionNumber from 'react-native-version-number';
+
+/**
+ * Send error to Sentry
+ */
+export const sentryReportError = (error) => {
+  Sentry.init({
+    dsn: 'https://c1ebae9159f741e8937abdbfbeba8e8a@o239606.ingest.sentry.io/5304101',
+  });
+  Sentry.withScope((scope) => {
+    scope.setExtra('App version', JSON.stringify(VersionNumber));
+    Sentry.captureException(error);
+  });
+};
+
+/**
+ * Copy access data from storage to the clipboard.
```

Comment:
It is really dangerous to do it. I agree it may really help identify some issues, but we cannot simply copy to the clipboard because many users may share this information, and without a strong password, they seed may be compromised.

Maybe we can copy the keys, content type and length, but never the content itself.

Reply:
Yeah, I agree with you. What about if we send to Sentry (if the user reports the error) this information you listed (the keys, content type and length)? I think this would already help us debug many cases and we are not sharing any sensitive data. And we could remove the 'Copy data' button.

Reply:
That's a great idea!

Reply:
Added the keys/content type/length on sentry report.